src/components/Media.js

```diff
@@ -573,10 +576,58 @@ export default class Video extends Component {
         >
           {this.renderChildren()}
         </video>
-      </div>
+      );
+    }
+    return (
+      <audio
+        className={classNames('video-react-video', this.props.className)}
+        id={videoId}
+        crossOrigin={crossOrigin}
+        ref={c => {
+          this.video = c;
+        }}
+        muted={muted}
+        preload={preload}
+        loop={loop}
+        playsInline={playsInline}
+        autoPlay={autoPlay}
+        poster={poster}
```

Comment:
Almost-blocking feedback. As you've specified elsewhere, `audio` elements cannot receive a `poster` image.

```suggestion
```